.github/workflows/tacos_unlock.yml

```diff
@@ -95,6 +95,11 @@ jobs:
       - name: Setup
         uses: ./tacos-gha/.github/actions/setup
 
+      - name: Override USER
+        if: github.event.action == 'closed'
+        run: |
+          gha-set-env 'USER' <<< ${{github.event.pull_request.user.login}}
```

Comment:
I had expected this to be pushed a couple levels deeper, into set-user-and-hostname...

https://gist.github.com/bukzor/6e4c35dee01690a496b136875abdcd01

but perhaps it's smarter to make this behavior be the default

https://gist.github.com/bukzor/873e19e41bf756093559b72c8a9a3b0a

Reply:
used the first behavior, as this is (for now) exclusive to unlock.